packages/main/src/components/AnalyticalCardHeader/AnalyticalCardHeader.jss.ts

```diff
@@ -8,10 +9,11 @@ const styles = {
     color: ThemingParameters.sapTextColor
   },
   cardHeader: {
-    paddingTop: '1rem',
-    paddingBottom: '1rem',
+    ...spacing.sapUiContentPadding,
     outlineOffset: '-0.125rem',
-    borderBottom: `0.0625rem solid ${ThemingParameters.sapTile_BorderColor}`,
+    boxShadow: ThemingParameters.sapContent_HeaderShadow,
+    //TODO use @sapUiTileSeparatorColor when available
+    borderBottom: `0.0625rem solid ${ThemingParameters.sapToolbar_SeparatorColor}`,
```

Comment:
actually this color should be `sapTile_SeparatorColor` but it is now available yet.